sdk/monitor/azure-monitor-query/README.md

```diff
@@ -234,6 +253,67 @@ for metric in response.metrics:
             print(metric_value.time_stamp)
 ```
 
+### Handling the response for metrics
+
+The metrics query API returns a `MetricsResult` object. This object has a list of `Metric` type along with other params like `interval`, `namespace` and `timespan`.
```

Comment:
also not sure about the context, but when you say the object `has` a list of objects + params, it seems a little unclear. By has, do you mean the `MetricsResult` object has those properties/attributes (instead of params)? So something more like the following?:
```suggestion
The metrics query API returns a `MetricsResult` object. This `MetricsResult` object has multiple properties, including: a list of `Metric` type objects, `interval`, `namespace`, and `timespan`.
```